pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -170,70 +81,86 @@ buildPythonPackage {
     zlib
   ];
 
-  propagatedBuildInputs = [
-    filelock
-  ];
+  propagatedBuildInputs = [ filelock ];
+
+  postPatch = let
+    # Bash was getting weird without linting,
```

Comment:
RE: cosmetic changes

Sorry I post this after the fact, but it's conventional  to keep postPatch next to src

Reply:
I always did pname -> version -> src -> patches -> nativeBuiltInputs -> buildInputs -> cmakeFlags -> post/pre/phase.
Where is this convention?

Reply:
What I think I remember is a comment from SuperSandro, but I'm failing to find it. The logic was, roughly speaking, to list attributes in the "chronological" order of when they are relevant: the first phase is unpacking the sources, so `src` goes first, and the very next thing is patching, and the very next after that is `postPatch`, and all of that happens before the configurePhase and before the build

Reply:
I see. Are native/build/propagated/inputs and cmakeFlags after patchPhase?

Reply:
Something like https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887?u=sergek:

>  `# Phases attrs are ordered exactly as they're executed in setup.sh`

Reply:
But also generally big formatting changes are undesirable because they're inconvenient to review, especially when formatting changes are entangled with functional changes